.travis.yml

```diff
@@ -62,6 +65,16 @@ base_install: &base_install |-
 
   [ "$TRAVIS_PULL_REQUEST" != "false" ] || export PYTEST_SENTRY_ALWAYS_REPORT=1
 
+install_node: &install_node |-
+  command -v volta && exit 0
+  wget --quiet "https://github.com/volta-cli/volta/releases/download/v$VOLTA_VERSION/volta-$VOLTA_VERSION-linux-openssl-1.0.tar.gz"
+  sudo tar -xzf "volta-$VOLTA_VERSION-linux-openssl-1.0.tar.gz" -C /usr/local/bin
+  # Running `volta -v` triggers setting up the shims in VOLTA_HOME (otherwise node won't work)
+  volta -v
+  # Running `node -v` and `yarn -v` triggers Volta to install the versions set in the project
```

Comment:
Might be good to provide a link to that issue you mentioned so we can replace this magic in the future.

Reply:
Sure, I'll link it here as well so they can see it. https://github.com/volta-cli/volta/issues/653#issuecomment-628909923